lib/kafka_ex/protocol/metadata.ex

```diff
@@ -1,5 +1,6 @@
 defmodule KafkaEx.Protocol.Metadata do
   alias KafkaEx.Protocol
+  import KafkaEx.Protocol.Common
```

Comment:
I'm not a big fan of `import`, but if you guys disagree then I won't put up much of a fight.